packages/validator/README.md

```diff
@@ -0,0 +1,96 @@
+# `@airnode/validator`
+
+Package for json specifications validation. Validation process can be configured with validator templates.
+
+## Usage
+
+The validator can be run as an NPM script, by providing template of specification, and the path to the JSON file of specification that will be validated:
+```sh
+npm run validate --template="[template]" --specs="[specsFile]"
```

Comment:
changed in 9e2fcc5

Reply:
Switching to yarn would bering quite a bit of overhaul to this, would it have some considerable benefits? Because otherwise I would keep it as npm

Reply:
> What is that commit part of? I still see this unchanged in this PR.

Yeah I had to switch it back, because I realized there are issues with arguments. It will stay as npm at least for this PR, if it would be causing issues, it can be rewritten to be using yarn in different PR.

Reply:
Mhh, strange. What was those issues?

Reply:
npm automatically parses arguments, which are then accessible as globals. It doesnt seem yarn does this, so there is need to parse it manually or add another library...

Reply:
I am pretty sure it work (I have used yarn to run scripts with arguments) and it's also what they say in https://classic.yarnpkg.com/en/docs/cli/run/#toc-yarn-run-script

Reply:
Not sure what I should look for in the link, but let me explain the issue in more detail. If I run following command `npm run validate --template="dir/template.json"`, all I have to do to access the `template` argument in the validator is `process.env.npm_config_template`. When running `yarn run validate --template="dir/template.json"`, the value of `template` argument obviosly won't be in npm global variable, but afaik needs to be parsed from `process.argv` either manually or using a library. Thats why using npm in this package is more viable, but if consistency is a problem I can later look for a simpler library then yargs that would do the job.

Reply:
Oh, I see. Npm parses the named parameters for you and yarn does not. 

We definitely should not rely on people using npm. Most people use npm and yarn interchangeably, so I agree with what @amarthadan was saying initially and migrate to yargs. (But it's OK to be done in a separate PR)